system-tests/fixtures/api/ComplexApi/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "author": "blemair",
+  "author": "f0b00n7",
```

Comment:
pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as `/Users/blemair/Code/`?